src/DataWrangling/JRA55.jl

```diff
@@ -444,7 +444,7 @@ function JRA55_field_time_series(variable_name;
     close(ds)
 
     N = (Nrx, Nry)
-    H = min.(N, (3, 3))
+    H = min.(N, (7, 7))
```

Comment:
```suggestion
    H = map(min, (N, N), (7, 7))
```

try not to use broadcasting unless you need it